dashboard/config/routes.rb

```diff
@@ -271,6 +271,16 @@
     end
   end
 
+  get '/course/:course_name', to: redirect('/courses/%{course_name}')
+  get '/courses/:course_name/vocab/edit', to: 'vocabularies#edit'
+
+  resources :courses, param: 'course_name' do
+    get 'vocab', to: 'courses#vocab'
+    get 'resources', to: 'courses#resources'
+    get 'code', to: 'courses#code'
+    get 'standards', to: 'courses#standards'
```

Comment:
this way is the railsiest:
```suggestion
    collection do
      get 'vocab'
      get 'resources'
      get 'code'
      get 'standards'
    end
```
the principle leading me to this conclusion is "convention over configuration", in this case meaning to avoid specifying the `to:` param unless it is deviating from the convention.

Reply:
This was actually member instead of collection because we need the course_name but it did clean up other things which was great!